theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -343,7 +348,7 @@ def fn(x):
                 assert any([isinstance(node.op, AveragePoolGrad)
                             for node in fc.maker.fgraph.toposort()])
             c_out = fc(data)
-            utt.assert_allclose(c_out, g_out)
```

Comment:
revert back that change. The old code is equivalent, but give a better error message.